h2o-py/h2o/estimators/gam.py

```diff
@@ -858,6 +865,22 @@ def remove_collinear_columns(self, remove_collinear_columns):
         assert_is_type(remove_collinear_columns, None, bool)
         self._parms["remove_collinear_columns"] = remove_collinear_columns
 
+    @property
+    def splines_non_negative(self):
+        """
+        Valid for I-spline only.  True if the I-splines are non-decreasing and False if the I-splines are non-
+        increasing.  If specified, must be the same size as gam_columns.  Values for bs != 2 will be ignored.  Default
```

Comment:
What does `values for bs != 2 will be ignored` mean?

Can we reword it to something like, "this parameter is only valid for monotone splines (bs = 2)`?

Reply:
It meant that bs spline is not i-spline.

Reply:
Good point.  I will add your suggestion.